clash-lib/src/Clash/Edalize/Edam.hs

```diff
@@ -303,10 +305,10 @@ pyField n x = hcat [squotes (pretty n), colon, space, x]
 pyDocField :: Text -> Text -> Doc ann -> Doc ann
 pyDocField n d x = vsep [pyComment d, pyField n x]
 
-relPath :: FilePath -> Doc ann
-relPath x = hcat
-  [ "os.path.relpath"
-  , parens (hsep $ punctuate comma [squotes (pretty x), "work_root"])
+joinPath :: FilePath -> Doc ann
+joinPath x = hcat
+  [ "os.path.joinpath"
```

Comment:
This function doesn't exist (at least in the python version on my work computer). Did you mean `os.path.join` or `joinpath` from `pathlib`?

Reply:
I meant `join`